### PR TITLE
Refs [TASK] [36142] Init base project

### PR DESCRIPTION
**Pull request by hhd0307**

### CODE REVIEW

File: app/src/main/java/com/sun/hero_01/extensions/FragmentExt.kt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        replace(id, fragment)
+        addToBackStack(null)
+    }
+}
+
+fun Fragment.removeFragment(fragment: Fragment) {
+    fragmentManager?.inTransaction { remove(fragment) }
+}
+
+fun Fragment.upperString(id: Int) = getText(id).toString().toUpperCase(Locale.getDefault())
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/hero_01/extensions/FragmentExt.kt]: enter end line

File: app/src/main/java/com/sun/hero_01/ui/MainActivity.kt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.sun.hero_01.ui
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import com.sun.hero_01.R
+
+class MainActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/hero_01/ui/MainActivity.kt]:
```suggestion
class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
```

File: app/src/main/java/com/sun/hero_01/ui/SplashScreenActivity.kt [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package com.sun.hero_01.ui
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+
+class SplashScreenActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/hero_01/ui/SplashScreenActivity.kt]:
```suggestion
class SplashScreenActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
```

File: app/src/main/java/com/sun/hero_01/ui/SplashScreenActivity.kt [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+        super.onCreate(savedInstanceState)
+        val intent = Intent(this, MainActivity::class.java)
+        startActivity(intent)
+        finish()
```
Comment (datdt-1935) [inline app/src/main/java/com/sun/hero_01/ui/SplashScreenActivity.kt]: đọc cái này nhé
https://github.com/framgia/coding-standards/blob/master/eng/android/coding_convention_android_kotlin.md#25-arguments-in-fragments-and-activities

File: app/src/main/res/values/colors.xml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="purple_200">#FFBB86FC</color>
+    <color name="purple_500">#FF6200EE</color>
+    <color name="purple_700">#FF3700B3</color>
+    <color name="teal_200">#FF03DAC5</color>
+    <color name="teal_700">#FF018786</color>
+    <color name="black">#FF000000</color>
+    <color name="white">#FFFFFFFF</color>
```
Comment (datdt-1935) [inline app/src/main/res/values/colors.xml]: 2 màu này có sẵn rồi. @android:color/black

**Review by datdt-1935 — APPROVED** [review]

**Pull request merged**